common/lib/xmodule/xmodule/js/js_test.yml

```diff
@@ -59,12 +59,18 @@ lib_paths:
     - common_static/js/src/utility.js
     - public/js/split_test_staff.js
     - common_static/js/src/accessibility_tools.js
+    - common_static/js/vendor/moment.min.js
 
 # Paths to spec (test) JavaScript files
 spec_paths:
     - spec/helper.js
     - spec
+    - spec/main_requirejs.js
 
 # Paths to fixture files (optional)
 fixture_paths:
     - fixtures
+
+requirejs:
```

Comment:
Can you explain this change. There is already a separate RequireJS-based test file called js_test_requirejs.yml. My understanding was that it was necessary to split the two files because of the way that the files are discovered. Does this change allow the two files to be merged back to one? Why was this necessary?